instrumentation/servlet/servlet-common/bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/servlet/ExperimentalSnippetHolder.java

```diff
@@ -5,12 +5,16 @@
 
 package io.opentelemetry.javaagent.bootstrap.servlet;
 
+import io.opentelemetry.instrumentation.api.internal.ConfigPropertiesUtil;
 import java.util.concurrent.atomic.AtomicReference;
 
 public class ExperimentalSnippetHolder {
 
   private static final AtomicReference<String> snippet =
-      new AtomicReference<>(System.getProperty("otel.experimental.javascript-snippet", ""));
+      new AtomicReference<>(
+          ConfigPropertiesUtil.getString("otel.experimental.javascript-snippet") == null
+              ? ""
+              : ConfigPropertiesUtil.getString("otel.experimental.javascript-snippet"));
```

Comment:
may be nicer to extract this to a static method, then you don't need to call `ConfigPropertiesUtil.getString()` twice